fastapi_sqla/sqla.py

```diff
@@ -311,4 +332,5 @@ def paginate(query: DbQuery, scalars=True) -> Page:
         return default_dependency
 
 
-Paginate: PaginateDependency = Pagination()
+Paginate = Annotated[PaginateDependency, Depends(Pagination())]
+Session = Annotated[SqlaSession, Depends(SessionDependency())]
```

Comment:
TBD, but I might be able to make this non breaking by using this form instead. But I feel that since the upgrade path is very simple, it's a good time to introduce a breaking change. We could also get rid of a couple legacy things.

```suggestion
Session: SqlaSession = SessionDependency()
```